fs/accounting/stats.go

```diff
@@ -624,17 +624,28 @@ func (s *StatsInfo) RemoveTransfer(transfer *Transfer) {
 	s.mu.Unlock()
 }
 
-// PruneTransfers makes sure there aren't too many old transfers
-func (s *StatsInfo) PruneTransfers() {
+// pruneTransfers removes transfers that already finished
+//
+// Must be called with the lock held
+func (s *StatsInfo) pruneTransfers() {
```

Comment:
Isn't it that this function should keep 100 last transfers by default?

Reply:
It does since it removes single transfer (note break in loop) and it's called once one of the transfer finishes. During applying your comment below I noticed it can't be used in `clear` because of it and I'm preparing patch as we speak.